posts/inside-rust/2024-05-09-rust-leads-summit.md

```diff
@@ -0,0 +1,77 @@
+* Alice Ryhl, Tokio maintainer, who spoke about the Rust for Linux Kernel effort and its use in Android.
+
+The schedule included a mix of scheduled roundtable discussions, invited talks, and a handful of "unconference" discussions whose topics were decided at the summit.
+
+## Why did it happen?
+
+The Rust Project has not had an in-person gathering that spans across teams since the last Rust All Hands at Mozilla Berlin in early 2019. Since then there have been major changes to Rust. The Rust Foundation was formed in early 2021 and now has a staff of ten. The Rust Core Team was dissolved in favor of a new Leadership Council. The open source project has grown and many new faces have joined its leadership ranks.
+
+The Rust Project itself has also grown, which has made the prospect of another Rust All Hands less feasible. Holding a smaller event focused on Project leadership allowed us to bring people together from across the project on a much smaller budget.
+
+## Key Takeaways
+
+One of the primary goals of the summit was building trust: between leads, between different parts of the project, and between the project and the foundation. By many reports, this was a smashing success. Many of the attendees had not met face-to-face before, and there's something about sitting in a room and talking about common interests that isn't easily replicated online.
+
+There were productive discussions around a number of topics, including how the Rust Project can better leverage the Foundation's resources to support its work. A full list of sessions is presented below.
+
+Finally, it's said that the first step to finding a solution is identifying the problem. There was broad agreement around some of the biggest challenges the Rust project faces, like its ability to make project-wide decisions. The group built consensus around [Project Goals] as a way to make some important kinds of decisions.
+
+By design, the leads summit was not a place for making final decisions – the Rust Project already has mechanisms like RFCs for that. However, it was intended as a place to share ideas and build consensus that will be used to accelerate future RFCs.
+
+## Sessions
+
+- Shape of the Project
+- Decision making process
+- Rust for Linux
+- 2023 Rust Survey
+- Leveraging Foundation resources
+- Achieving the Foundation's mission
+- The purpose, structure, and workings of teams
+- Project goals
+- Team-building sharing session
+
+## Looking forward
+
+All of the participants surveyed said that the leads summit served a needed purpose for the project. We hope there will be more events like the leads summit in the future, and that they can grow to include more of the Rust project and its leadership.
+
+For this event to be valuable, it's important for us to follow through on the ideas that we talked about. We've already seen one RFC on [Project Goals] and hope to see similar RFCs in the coming months.
```

Comment:
```suggestion
For this event to be valuable, it's important for us to follow through on the ideas that we talked about. We've already seen one RFC on [Project Goals] and hope to see similar RFCs in the coming months. The Leadership Council has also started offering experimental travel funding to project members.
```

Reply:
We can leave this out or not, but this is another thing that we built a lot of momentum around at the summit and have put into effect since then.

Reply:
I'm ok with it